package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "scripts": {
     "build": "webpack --mode production",
-    "watch": "webpack --watch",
+    "dev": "webpack --watch --progress --mode development",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
- Kindly fix your scripts in your package.json. I can't build the project and start the server.
![Screen Shot 2021-08-18 at 19 33 06](https://user-images.githubusercontent.com/22258274/129989422-6fa15b09-8a8d-49c3-96da-ae6c6c93890f.png)

- Tried to follow the steps described in your README file, but nothing happens.

Reply:
**All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.**

_More information_[ Here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md).